### ISSUE TITLE
Scribing Tools and Scroll not on table

### ISSUE THREAD

**Issue by AzraelTWC** [Azanor/thaumcraft-beta#876]
Not sure if this has been reported but I checked a few pages and couldn't 'find' it. The models don't show up on the table at all. 

Also this could be time to suggest returning to the 1x2 table via placing 2 research tables next to each other and having them combine. Just as a aesthetic addition, but bugs and new content are priority for sure.

**Comment by Geethebluesky** [comment]
Are you using Beta 15 while seeing this?

**Comment by AzraelTWC** [comment]
It seems to be an issue ever since 1.12.2 was released. Source: Watching TC6 guide videos the scribing tools always seem to be missing.

but yes in B15 they are missing for sure.

**Comment by Geethebluesky** [comment]
Huh. I never noticed they were gone before you mentioned it. Perhaps this is intentional or hasn't been added back in yet because it's just aesthetic.

**Comment by AzraelTWC** [comment]
Its defiantly worth the time and is removed either because of a bug or temporary for some reason. This mod is all about the game play and aesthetic. Its not as important as getting all the features in and polished, but it is certainly worth it. Anything that helps bring the player and add immersion is important and I think Azanor strides for.

**Comment by Geethebluesky** [comment]
Fair enough, people get immersion from all sorts of different things after all.
